deploy-web/src/components/sdl/PlacementFormModal.tsx

```diff
@@ -180,7 +159,7 @@ export const PlacementFormModal: React.FunctionComponent<Props> = ({ control, se
                       <TextField
                         type="number"
                         variant="outlined"
-                        label="Pricing"
+                        label={`Pricing, ${toReadableDenom(currentService.placement.pricing.denom)}`}
```

Comment:
This would add more clarity as now the token select is in the parent component

Reply:
Yes good idea.